pyproject.toml

```diff
@@ -160,11 +156,10 @@ lint.per-file-ignores = {"__init__.py" = [
     "ANN201", # TODO: issing-return-type-undocumented-public-function
     "ANN202", # TODO: missing-return-type-private-function
     "D100", # undocumented-public-module
-    "D103", # TODO: undocumented-public-function
```

Comment:
I think it should be okay to not have docstrings in the test functions

Reply:
I do think it's something we should work towards